modules/learn/pages/clusters-and-availability/groups.adoc

```diff
@@ -1,6 +1,7 @@
 = Server Group Awareness
```

Comment:
There is a para here that should be removed as it is no longer true. It is the following one:

"Protection is optimal when groups are assigned equal numbers of nodes, and replica vBuckets and index replicas are thereby distributed such that none ever occupies the same group as its associated active vBucket or index. By contrast, when groups are not assigned equal numbers of nodes, rebalance can only produce a best effort redistribution of replicas: this may result in one or more replica vBuckets and/or index replicas each occupying the same group as their associated active vBuckets or indexes; meaning that vBucket data and/or index data may be lost if such a group becomes unavailable."

Reply:
Also the following bullet should be removed:
"Failover should be enabled for server groups only if three or more server groups have been established, and sufficient capacity exists to absorb the load of any failed-over group."

Reply:
This bullet also should be removed:
"If a cluster contains only one node, or only one group of nodes, enabling Group Failover has no effect; meaning that failover can only be performed on a per-node basis."

Reply:
Reword "For information on failing over individual nodes, see [Fail a Node over and Rebalance]" to 
"For information on failover see [Failover and Rebalance]" with same link